packages/salesforcedx-vscode-apex/test/jest/views/settings/configurations.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
+
+describe('Should synchronize init jobs in server by default', () => {
+  const sandbox = sinon.createSandbox();
+  beforeEach(() => {
+    sandbox.stub(vscode.workspace, 'getConfiguration').returns({
+      get: () => true
+    });
+  });
+
+  afterEach(() => sandbox.restore());
+
+  it('should enable enableSynchronizedInitJobs by default', () => {
+    const enableSyncInitJobs: boolean = vscode.workspace
+      .getConfiguration()
+      .get<boolean>('salesforcedx-vscode-apex.wait-init-jobs', false);
+
+    expect(enableSyncInitJobs).toBe(true);
+  });
+
+  it('value of enableSynchronizedInitJobs should be updated after changed', () => {
+    sandbox.restore();
+    sandbox.stub(vscode.workspace, 'getConfiguration').returns({
+      get: () => false
+    });
+    const enableSyncInitJobs: boolean = vscode.workspace
+      .getConfiguration()
+      .get<boolean>('salesforcedx-vscode-apex.wait-init-jobs', true);
+
+    expect(enableSyncInitJobs).toBe(false);
+  });
```

Comment:
I think these tests are going to be something you do on settings.test.ts after you move that logic out of the languageServer.  Although in that case you'll be mocking the vscode.workspace.getConfuration.get response to just ensure it does what we expect with what vscode gives us back (note all of these are mocked responses in jest unit tests)